application/F3DOptionsParser.cxx

```diff
@@ -359,10 +361,10 @@ void ConfigurationOptions::PrintHelp(cxxopts::Options& cxxOptions)
   // clang-format off
   const std::vector<std::pair<std::string, std::string> > examples =
   {
-    { "f3d file.vtu -xtgans", "View a unstructured mesh in a typical nice looking sciviz style" },
-    { "f3d file.glb -tuqap --hdri=file.hdr", "View a gltf file in a realistic environment" },
-    { "f3d file.ply -so --point-size=0 --comp=-2", "View a point cloud file with direct scalars rendering" },
-    { "f3d folder", "View all files in folder" },
+    { this->ExecutableName+" file.vtu -xtgans", "View a unstructured mesh in a typical nice looking sciviz style" },
+    { this->ExecutableName+" file.glb -tuqap --hdri=file.hdr", "View a gltf file in a realistic environment" },
+    { this->ExecutableName+" file.ply -so --point-size=0 --comp=-2", "View a point cloud file with direct scalars rendering" },
+    { this->ExecutableName+" folder", "View all files in folder" },
```

Comment:
add spaces: ` + `

But I would have expected clang format to grab that.

Reply:
There's a `// clang-format off` just before preventing it

Reply:
right

Reply:
Letting clang handle it gives:
```cpp
  const std::vector<std::pair<std::string, std::string> > examples = {
    { this->ExecutableName + " file.vtu -xtgans",
      "View a unstructured mesh in a typical nice looking sciviz style" },
    { this->ExecutableName + " file.glb -tuqap --hdri=file.hdr",
      "View a gltf file in a realistic environment" },
    { this->ExecutableName + " file.ply -so --point-size=0 --comp=-2",
      "View a point cloud file with direct scalars rendering" },
    { this->ExecutableName + " folder", "View all files in folder" },
  };
```

Should I just add the spaces or remove the `// clang-format` comments?

Reply:
yes, formatting is acceptable with clang format imo